### DISCUSSION TITLE
禁言相关的api好像有点问题的样子

### DISCUSSION THREAD

**Discussion by jiayuqi7813** [Excaive/miraicle#12] [Q&A]
代码如下
`import miraicle

@miraicle.Mirai.receiver('GroupMessage')
def Dragonmeet(bot: miraicle.Mirai, msg: miraicle.GroupMessage):
    if msg.plain=='禁言':
        bot.mute_all(group=msg.group)`

图片这样写完后发送禁言没有任何反应，我这边进行了一下输出返回的是4000
![image](https://user-images.githubusercontent.com/63686458/132195325-d389dbf1-31ab-493c-a913-ee4938a85abd.png)

然后我这里进行了一个抓包，发现POST请求发送的是get方式的数据，而不是官方api的json，是我操作方式问题吗，还是什么情况
![image](https://user-images.githubusercontent.com/63686458/132195698-33d0a06b-2870-4a6e-8711-976f5e30c455.png)

**Comment by Excaive** [comment]
当时我写的时候确实没对禁言相关的 api 做测试，已经在新版本（0.3.8）中修复了

**Reply by jiayuqi7813** [reply]
辛苦，然后我发现撤回好像也不能用（，问题一样是400

**Reply by Excaive** [reply]
修复了，刷了一波版本号
